app/servicies/foreman_statistics/trend_importer.rb

```diff
@@ -0,0 +1,63 @@
+module ForemanStatistics
```

Comment:
typo in folder name? should be services?

Reply:
oh :facepalm: